mlflow/sagemaker/container/scoring_server/serve.py

```diff
@@ -1,73 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This file is unused and duplicates code present here: https://github.com/mlflow/mlflow/pull/395/files#diff-fe6d2b97b3f89591586a279574340099R72